### DISCUSSION TITLE
Clarification needed: Setting up 'frame' directory in make_video.sh script from OpenDA documentation

### DISCUSSION THREAD

**Discussion by carbonatezero** [OpenDA-Association/OpenDA#294] [General]
The script linked [here](https://github.com/OpenDA-Association/OpenDA/blob/dc5f23806c900b41cbe8a9f4514cfeaf06cc3b5d/course/exercise_double_pendulum_part1/make_video.sh#LL7C40-L7C46) attempts to make a video but there is some confusion about the `frame` directory. It would be helpful to have more detailed instructions on how to set up the `frame` directory and what its contents should be.

**Comment by nilsvanvelzen** [comment]
This script is not part of the course itself but is used to create a video of the running model for illustrative purpose. In #295 I have added some comments in the code of two scripts to clarify it a bit.
